apps/api/src/friendships/friendships.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller, Post, Delete, Body, Get, Query, UseGuards } from "@nestjs/common";
+import { FriendshipsService } from "./friendships.service";
+// import { JwtAuthGuard } from '../auth/jwt-auth.guard';
+import { User } from "types";
+
+//@UseGuards(JwtAuthGuard)
+@Controller('friendships')
+export class FriendshipsController {
+    constructor(
+        private readonly fiendshipsService : FriendshipsService
+    ) {}
+
+    @Post()
```

Comment:
This should be a PUT I believe. initiator_id can be retrieved with decorator @User in common (needs to implement JWTGuard), and target_id as a param. Can be fixed post-merge

Reply:
Solved